proto/common/v1/ipns.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package sonrio.common.v1;
+
+option go_package = "github.com/sonr-io/sonr/thirdparty/types/common";
```

Comment:
`thirdparty` should be `third_party` to match conventions

Reply:
just what i was thinking folder was `thirdparty` before didnt want to screw with it but will do

Reply:
Fixed